app/views/layouts/_notification_banner.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-!-margin-bottom-0">
+    <%= render(GovukComponent::NotificationBannerComponent.new(
+    title_text: flash[:success] != false ? t(".success") : t(".alert"),
+    success: flash[:success] != false,
+  )) do %>
+    <h3 class="govuk-heading-m"><%= flash[:heading] %></h3>
```

Comment:
We should not be using a `<h3>` tag here.

Sorry 🙈 I thought we'd fixed this in a previous PR, because it was part of the accessibility work. And now I'm just confused.

Reply:
Oh, I've just checked. It looks like I raised it here on a different PR https://github.com/DFE-Digital/itt-mentor-services/pull/1023#discussion_r1763616749 

So my bad. We never actually did fix it for the main `application.html.erb` layout.

Reply:
So, we want to use `nb.with_heading` here – a bit like this:

https://github.com/DFE-Digital/itt-mentor-services/blob/d242b2408286606a265f3bb05189b2bd51801137/app/views/placements/schools/placements/preview.html.erb#L10-L12

I haven't tried it, but I think something like this should work:

```ruby
<%= render(GovukComponent::NotificationBannerComponent.new(
  title_text: flash[:success] != false ? t(".success") : t(".alert"),
  success: flash[:success] != false,
)) do |nb| %>
  <% nb.with_heading(flash[:heading]) %>
  <% if flash[:body].present? %>
    <p class="govuk-body"><%= flash[:body] %></p>
  <% end %>
<% end %>
```